CHANGELOG.md

```diff
@@ -2,464 +2,292 @@
-- Base Appliance
-  - cloud-init rpm added (but there are root ssh login issues)
-  - iptables configured via kickstart 
+- Foreman
+  - Filtering of available Configuration Profiles based on selected Configured Systems in provisioning dialog
+  - Exposed additional properties to reporting
+- Automate: Ability to override request description with user message
 
-### [Removed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+12+Ending+Apr+20%2C+2015%22+label%3A%22technical+debt%22)
+### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+23+Ending+May+11%2C+2015%22+label%3Aenhancement) 
 
-- SmartProxy: removed from UI
-- IP Address Form Field: Removed from UI (use Hostname)
+- UI: Foreman Enhancements
 
 
-## Unreleased - as of Sprint 21 end 2015-03-30
+# Botvinnik Release
 
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+21+Ending+Mar+30%2C+2015%22+label%3Aenhancement)
+### Added Features
 
-- Providers: OpenStack
+#### Providers
+- General
+  - Added refresh status and errors, viewable on Provider Summary Page.
+  - Added collection of raw power state and exposed to reporting.
+  - Orchestration: Support for Retirement
+  - Update authentication status when clicking validate for existing Providers
+  - Hosts and Providers now default to use the hostname column for communication instead of IP address.    
+- SCVMM
+  - Provisioning from template, Kerberos Authentication
+  - Virtual DVD drives for templates
+- Containers: UI updates, refresh button
+- Kubernetes 
+  - Inventory Collection
+  - EMS Refresh scheduling
+- Foreman
+  - Provider refresh
+  - Enabled Reporting / Tagging
+  - Exposed Foreman models as Automate service models
+  - Zone enablement
+  - EMS Refresh scheduling
+  - Added tag processing during provisioning
+  - Added inventory collection of direct and inherited host/host-group settings
+  - Organization and location inventory
+- Cloud Providers
+ - Cloud images and instances: Added root device type to summary screens.
+ - Cloud flavors: Added block storage restriction to summary screens.
+ - Enabled Reporting
+- OpenStack
   - Inventory for Heat Stacks (Cloud and Infrastructure)
   - Connect Cloud provider to Infrastructure provider
-- Providers: General
-  - EMS Refresh scheduling for Foreman and Kubernetes
-- Rest API
-  - Providers CRUD
-  - Refresh via /api/providers
-- Provisioning
-  - Heat Orchestration provisioning through services
-  - Foreman
-     - Provisioning of bare metal systems
-     - Organization and location inventory
-     - Uses latest Foreman Apipie gem
-- Automate: Enhanced UI import
-- Fleecing
- - Qcow3 support
- - VSAN support (VMware)
- - OpenStack instances
-- UI
- - Foreman includes provisioning
+  - OpenStack Infrastructure Host Events
+  - Autoscale compute nodes via Automate
+  - Support for non-admin users to EMS Refresh
+  - Tenant relationships added to summary screens
+  - Undercloud Event processing
```

Comment:
Yeah ... Six of one, half-dozen of the other, imo.  However, I guess since it's not actually grouped under "OpenStack" anything here, then @Fryguy is right.